plugins/producer_plugin/test/test_read_only_trx.cpp

```diff
@@ -50,7 +50,8 @@ auto make_unique_trx( const chain_id_type& chain_id ) {
 
 BOOST_AUTO_TEST_SUITE(read_only_trxs)
 
-void error_handling_common(std::vector<const char*>& specific_args) {
+// app_init_status: true -- succeeded, false -- failed
+void test_configs_common(std::vector<const char*>& specific_args, bool app_init_status) {
```

Comment:
Instead of using `bool` which is not very explicit, I prefer using `enum class`, like:

```suggestion
enum class app_init_status { failed, succeeded };
void test_configs_common(std::vector<const char*>& specific_args, app_init_status status) {
```